sympy/concrete/tests/test_sums_products.py

```diff
@@ -258,15 +258,15 @@ def test_geometric_sums():
     #issue 11642:
     result = Sum(0.5**n,(n, 1, oo)).doit()
     assert result == 1
-    assert result.is_Float
+    assert not result.is_Float
```

Comment:
Actually it was intentional since 0.5 was coerced to a rational. But this is easily made to work as earlier.